moped-editor/src/views/projects/projectView/ProjectPhases.js

```diff
@@ -48,122 +84,121 @@ const ProjectPhases = ({
-      lookup: phaseNameLookup(data),
-      validate: (row) => !!row.phase_id,
+      title: "Phase",
+      field: "moped_phase",
+      validate: (row) => !!row.moped_phase?.phase_id,
+      render: (row) => row.moped_phase?.phase_name,
       editComponent: (props) => (
-        <DropDownSelectComponent {...props} name={"phase_name"} data={data} />
+        <FormControl style={{ minWidth: 150 }}>
+          <Autocomplete
+            id="phase_id_autocomplete"
+            name="phase_id_autocomplete"
+            options={phaseOptions}
+            getOptionLabel={(phase) => phase.phase_name}
+            getOptionSelected={(option, value) =>
+              option.phase_id === value.phase_id
+            }
+            value={props.value || null}
+            onChange={(event, value) => {
+              return props.onChange(value);
+            }}
+            renderInput={(params) => (
+              <TextField {...params} autoFocus style={{ minWidth: 200 }} />
+            )}
+          />
+          <FormHelperText>Required</FormHelperText>
+        </FormControl>
       ),
-      width: "18%",
+      width: "25%",
     },
     {
-      title: "Sub-phase name",
+      title: "Subphase",
       field: "subphase_id",
-      lookup: subphaseNameLookup,
+      render: (rowData) => rowData.moped_subphase?.subphase_name,
       editComponent: (props) => (
         <DropDownSelectComponent
           {...props}
           name={"subphase_name"}
           data={data}
         />
       ),
-      width: "18%",
+      width: "20%",
     },
     {
       title: "Description",
       field: "phase_description",
-      width: "18%",
+      width: "25%",
     },
     {
-      title: "Start date",
+      title: "Start",
       field: "phase_start",
       render: (rowData) =>
         rowData.phase_start
           ? format(parseISO(rowData.phase_start), "MM/dd/yyyy")
           : undefined,
       editComponent: (props) => (
-        <DateFieldEditComponent
-          {...props}
-          name="phase_start"
-          label="Start Date"
-        />
+        <DateFieldEditComponent {...props} name="phase_start" label="Start" />
       ),
-      width: "18%",
+      width: "10%",
     },
     {
-      title: "End date",
+      title: "End",
       field: "phase_end",
       render: (rowData) =>
         rowData.phase_end
           ? format(parseISO(rowData.phase_end), "MM/dd/yyyy")
           : undefined,
       editComponent: (props) => (
-        <DateFieldEditComponent {...props} name="phase_end" label="End Date" />
+        <DateFieldEditComponent {...props} name="phase_end" label="End" />
       ),
-      width: "18%",
+      width: "10%",
     },
     {
       title: "Current",
       field: "is_current_phase",
       lookup: { true: "Yes", false: "No" },
+      render: (rowData) =>
+        rowData.is_current_phase ? (
+          <Chip label="Yes" color="primary" icon={<Check />} />
+        ) : (
+          "No"
+        ),
```

Comment:
I like the chip, but I would prefer it to be green.

Reply:
👍 
<img width="1071" alt="Screen Shot 2022-11-16 at 1 14 39 PM" src="https://user-images.githubusercontent.com/14793120/202260704-3be34a6e-91fc-4012-8062-7c7c73ff4b60.png">

Reply:
i don't love the conflicting colors with the status badge, but i agree the blue/primary color is confusing because we use it for action buttons. i'll ship it this way! 

<img width="1129" alt="Screen Shot 2022-11-16 at 1 16 41 PM" src="https://user-images.githubusercontent.com/14793120/202260908-df967521-4ba5-4a51-9a95-44233ef545c8.png">

Reply:
Oh gawd, that's too harsh. Perhaps without the Chip but with a green-check icon?

![Yes Chip](https://user-images.githubusercontent.com/5566727/202268383-f56146be-e349-4fe3-8742-5000d8ee8df0.png)

Reply:
how about we also hide "No" and leave the column empty?

<img width="1438" alt="Screen Shot 2022-11-17 at 11 45 01 PM" src="https://user-images.githubusercontent.com/14793120/202618831-37c48847-4722-437f-807e-a819c822dc27.png">

that also let's put the icon on the left which looks better imo.

Reply:
btw i pushed this to https://github.com/cityofaustin/atd-moped/pull/871

Reply:
Oooo, that's really nice! Hiding "No" really cleans it up. Agree the icon on the left look better now.